config/settings/roles.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.admin import UserAdmin
+from rolepermissions.roles import (
+    AbstractUserRole
+)
+
+
+class SystemAdmin(AbstractUserRole):
+    available_permissions = {
+        'drop_tables': True,
```

Comment:
we can have `can_manage_project_managers`, `can_manage_field_data`, `can_manage_reports`